### PR TITLE
add load shedding to federation grpc requests

### PR DESCRIPTION
**Pull request by fabiokung**
Protection for REST endpoints will come at a later date.

### CODE REVIEW

**Comment by tbak** [comment]
Looks good, but is missing REST API integration. For completeness we should cover that as well, as a single rouge client can still generate excessive load down to TitusMaster.

File: titus-server-federation/src/main/java/com/netflix/titus/federation/endpoint/EndpointConfiguration.java [hunk 1]
```diff
@@ -34,4 +34,7 @@
      */
     @DefaultValue("30000")
     long getGrpcServerShutdownTimeoutMs();
+
+    @DefaultValue("true")
+    boolean isAdmissionControllerEnabled();
```
Comment (tbak) [inline titus-server-federation/src/main/java/com/netflix/titus/federation/endpoint/EndpointConfiguration.java]: Add REST API integration.

**Comment by fabiokung** [comment]
@tbak I mention the REST API in the PR description.

**Comment by tbak** [comment]
@fabiokung Sorry, missed that.

**Pull request merged**